docs/rules/rules.tex

```diff
@@ -493,6 +498,8 @@ \subsection{Validation parameters}
   we have $x<y<z$. These subsets are defined in
   Section~\ref{sec:vss-phases}.
 
+  \improvement{This paragraph is pretty horrible to read, just because the
+    subscripts are so long.}
```

Comment:
There are soooo many names. That I opted for very long and descriptive names. But they are all macros, for the reason that this is also ugly. Let's keep this in mind.